src/js/utils/form-utils.jsx

```diff
@@ -156,3 +156,15 @@ export const validateFutureDate = (pickedDate) => {
   const today = moment(new Date());
   return date.startOf('day').isSameOrBefore(today.startOf('day'));
 };
+
+/**
+ * Method to check if first date is later than the second date
+ * @return true if the first date is later than the second date
+ * @param laterDate
+ * @param earlierDate
+ */
+export const validateOneAfterAnotherDate = (laterDate, earlierDate) => {
```

Comment:
`validateOneAfterAnotherDate` - "Method to check if first date is later than the second date",  but you check `isSameOrAfter`.